### PR TITLE
Improve quality of the camera JPEG images

### PR DESCRIPTION
**Pull request by gang65**

### CODE REVIEW

**Comment by filmo** [comment]
I would make camera mode a parameter of `__init__` with 'auto' being the default to maintain backward compatibility. Then the part could be called with 'sport' or any of the other listed options in the PiCamera documentation by users when they customize their car configuration.

Something like

`self.camera.exposure_mode = mode_parameter`

**Comment by gang65** [comment]
@filmo After intense testing, I have proven that with decreasing exposure time (by using `sports` mode), improve detection of shapes by NN. 
My intention is to enable it by default, to give end users the best possible settings out of the box.

Of course this setting need to be confirmed by other users.
If you want I could show some images with `auto` and `sports` mode.

Do you know why unit tests are failing?

File: donkeycar/parts/datastore.py [hunk 1]
```diff
@@ -193,13 +193,13 @@ def put_record(self, data):
...
                 name = self.make_file_name(key, ext='.jpg')
-                val.save(os.path.join(self.path, name))
+                val.save(os.path.join(self.path, name), format='jpeg', quality=90)
                 json_data[key]=name
 
             elif typ == 'image_array':
                 img = Image.fromarray(np.uint8(val))
                 name = self.make_file_name(key, ext='.jpg')
-                img.save(os.path.join(self.path, name))
+                val.save(os.path.join(self.path, name), format='jpeg', quality=90)
```
Comment (hertzi) [inline donkeycar/parts/datastore.py]: copy/paste error?

**Comment by wroscoe** [comment]
I'm closing this because I've moved the code for Tubs to this repo. https://github.com/autorope/donkeypart_tub

**Pull request closed without merging**